### ISSUE TITLE
Using Github Page

### ISSUE THREAD

**Issue by veggiemonk** [jackfranklin/the-refactoring-tales#7]
How about using github page (creating a branch named gh-page, which can be done automatically in the settings of your repo :smile: )

By the way, your talk on jspm was really help to get started to jspm! Nice work.

Cheers

**Comment by jackfranklin** [comment]
@veggiemonk yeah, that would be a better way of setting this up. I need to revisit this work at some point, both from a content POV (lots more I'd like to add!) but also making it easier to update and work with.
